views/jobs.blade.php

```diff
@@ -114,6 +114,15 @@
                                 <span class="font-semibold">{{ $job->attempt }}</span>
                             </div>
 
+                            @if(config('queue-monitor.ui.show_data'))
+                                @foreach($job->getData() as $key => $value)
+                                    <div class="text-xs">
+                                        <span class="text-gray-600 font-medium">{{ $key }}:</span>
+                                        <span class="font-semibold">{{ $value }}</span>
+                                    </div>
+                                @endforeach
```

Comment:
Since the data column can not only consist of string-keys and string-values I would display the data as json encoded string.
```suggestion
                                <textarea rows="4"
                                          class="w-64 text-xs p-1 border rounded"
                                          readonly>{{ json_encode($monitor->data, JSON_PRETTY_PRINT) }}</textarea>
```